projects/common-form-elements/src/lib/dynamic-multiple-dropdown/dynamic-multiple-dropdown.component.ts

```diff
@@ -33,8 +33,10 @@ export class DynamicMultipleDropdownComponent implements OnInit, OnChanges, OnDe
 
   _: any = _;
 
-
   public isDependsInvalid: any;
+  public isSearchables: any;
+  public showMaterSelect: boolean = true;
```

Comment:
can we make it `isSearchables` as `isSearchable` and also spell check `showMaterSelect`

Reply:
Done